inventree_kicad/serializers.py

```
@@ -285,15 +285,30 @@
+
+        except:
+            pass # ignore if there are any issues
+
+
+        # load the global visibility settings if available and valid
+        try:
+            kicad_global_field_visibility = self.plugin.get_setting('KICAD_FIELD_VISIBILITY_PARAMETER_GLOBAL',
+                                                                    None).split(',')
+
+            kicad_global_field_visibility = [field.strip().lower() for field in kicad_global_field_visibility]
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)